Dockerfile

```diff
@@ -1,32 +1,16 @@
-FROM centos:6.7
+FROM mozillabteam/bmo-base:slim
```

Comment:
I think this repo should probably be added to circle as well since it's a core component of the BMO app as a whole.